vms/platformvm/blocks/builder/builder.go

```diff
@@ -247,7 +253,7 @@ func (b *builder) dropExpiredStakerTxs(timestamp time.Time) {
 	minStartTime := timestamp.Add(txexecutor.SyncBound)
 	for b.Mempool.HasStakerTx() {
 		tx := b.Mempool.PeekStakerTx()
-		startTime := tx.Unsigned.(txs.Staker).StartTime()
+		startTime := tx.Unsigned.(txs.PreContinuousStakingStaker).StartTime()
```

Comment:
not sure if I should be defensive and guard the cast. Following continuous fork activation, all AddStakerTxs will not be among mempool staker txs.